presto-product-tests/README.md

```diff
@@ -143,6 +143,12 @@ presto-product-tests/bin/run_on_docker.sh <profile> -t com.facebook.presto.tests
 presto-product-tests/bin/run_on_docker.sh <profile> -t sql_tests.testcases.system.selectInformationSchemaTables
 ```
 
+Profile specific tests are tests which runs only for that particular profile. It confirms if the profile is same as what is claims.
```

Comment:
Please rewrite this section like so:

Tests belong to a single or possibly multiple groups. Java based tests are
tagged with groups in the `@Test` annotation and convention based tests
have group information in the first line of the test file. Instead of
running single tests, or all tests, users can also run one or more
test groups. This enables users to test features in a cross functional
way. To run a particular group, use the `-g` argument as shown:

```
# Run all tests in the string_functions and create_table groups
presto-product-tests/bin/run_on_docker.sh <profile> -g string_functions,create_tables
```

Some groups of tests can only be run on specific profiles. For example, all
tests that test Kerberos with HDFS impersonation can only be run on the
`singlenode-kerberos-hdfs-impersonation` profile. The group of tests
that must be run on the `singlenode-kerberos-hdfs-impersonation` profile
is also called `singlenode-kerberos-hdfs-impersonation`. The group is the
same as the profile name. This is true for the following group/profile
pairs: `singlenode-hdfs-impersonation`, `singlenode-kerberos-hdfs-impersonation`
and `singlenode-kerberos-hdfs-no-impersonation`. Tests that require
a specific profile to run are called profile specific tests. In addition
to their respective group, all such tests also belong to a parent group called
`profile_specific_tests`. Thus, when running tests on the `singlenode`
profile make sure to exclude all profile specific test by adding the
`profile_specific_tests` to the list of excluded groups. The examples
below illustrate the above concepts:

```
# Run the tests for Kerberos with HDFS impersonation
presto-product-tests/bin/run_on_docker.sh singlenode-kerberos-hdfs-impersonation -g singlenode-kerberos-hdfs-impersonation
# Run all tests but, among other groups, exclude all profile specific
# tests because the singlenode profile is used
presto-product-tests/bin/run_on_docker.sh singlenode -x quarantine,big_query,mysql_connector,postgresql_connector,profile_specific_tests
```